inst/assets/pkgdown.css

```diff
@@ -168,6 +172,29 @@ a.anchor {
   margin-bottom: 0.5em;
 }
 
+/* bootstrap-toc tweaks */
+nav[data-toggle="toc"] .nav > li > a {
+  font-size: 17px;
```

Comment:
This feels too big to me. Can we make it 1.1em? And not bold?

The left padding also feels a bit too big to me — I think the headings should be left aligned with contents. And where does the purple colour come from?

Reply:
Site with new tweaks at: https://boring-kare-88d4d5.netlify.com/

Reply:
Perhaps a happy medium with the adjustments below?
```css
font-weight: 500;
font-size: 1.4rem;
```
I think a _little_ more pop would help.

Reply:
With more pop: https://competent-feynman-455f34.netlify.com/news/index.html

Reply:
I like it! I just spent _way_ too long trying to figure out how to select the submenu items, and couldn't get it quite right (I am not a CSS pro by any measure), but I think those could be a scoch larger, if someone else knows how to scope it right (if not, nbd).